sources/packages/web/src/components/common/ScholasticStandingForm.vue

```diff
@@ -45,26 +72,58 @@ export default defineComponent({
     },
     processing: {
       type: Boolean,
-      required: true,
+      required: false,
       default: false,
     },
   },
   setup(props, context) {
+    const initialData = ref({} as ScholasticStandingSubmittedDetails);
+    const { dateOnlyLongString } = useFormatters();
+
+    watchEffect(async () => {
+      if (!props.initialData) {
```

Comment:
Since `scholasticStandingId` is optional, I would suggest checking it as below.
```ts
if (props.scholasticStandingId) {
  const applicationDetails =
      await ScholasticStandingService.shared.getScholasticStanding(
        props.scholasticStandingId,
      );
...
```